lib/ash/resource/attribute.ex

```diff
@@ -62,6 +64,11 @@ defmodule Ash.Resource.Attribute do
       default: true,
       doc: "Whether or not the value can be written to"
     ],
+    private?: [
+      type: :boolean,
+      default: false,
+      doc: "Whether or not the value can be accessed by extensions"
```

Comment:
I might say "Whether or not the attribute will appear in any interfaces created off of this resource, e.g `AshJsonApi` and `AshGraphql`"

Reply:
Updated.